stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -762,8 +761,7 @@ function generic_matmatmul(tA, tB, A::AbstractVecOrMat{T}, B::AbstractMatrix{S})
     generic_matmatmul!(C, tA, tB, A, B)
 end
 
-const tilebufsize = 10800  # Approximately 32k/3
```

Comment:
Apparenly ArrayLayouts.jl uses this... Can we keep it for legacy reasons (while also update ArrayLayouts to not use it?)

Reply:
I added and merged it right away.